src/systems/ec/implementations/vanilla/packages/ec-component-gallery/ec-component-gallery.js

```diff
@@ -223,10 +226,21 @@ class Gallery {
     } else {
       this.overlay.removeAttribute('open');
     }
+
+    // Focus item
+    this.selectedItem.focus();
```

Comment:
what about users closing the modal with esc key?

Reply:
it's already there